x/staking/app_test.go

```diff
@@ -39,6 +39,16 @@ func checkDelegation(
 	require.False(t, found)
 }
 
+func ExecuteNextEpoch(t *testing.T, app *simapp.SimApp) {
+	lastBlockHeight := app.LastBlockHeight()
```

Comment:
Should this be named "CurrentEpochStartHeight()"

Reply:
also can some docs be added to this function explaining what the state is before and after it runs. (Are all the queues clearead, etc?)

Reply:
It's built just for testing purposes. It fill dummy blocks (Epoch blocks count) from current height, not from epoch start height. And if we do that, it always contain epoch block. The thing is utility function for calculating CurrentEpochStartHeight is inside keeper as EpochInterval is not constant